xml/Accessibility/CAccPropServicesClass.xml

```diff
@@ -371,16 +371,16 @@
         <ReturnType>System.Void</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="pIDString" Type="System.Byte" RefType="ref" />
-        <Parameter Name="dwIDStringLen" Type="System.UInt32" />
-        <Parameter Name="phmenu" Type="System.IntPtr">
+        <Parameter Name="pIDString" Type="System.Byte" RefType="ref" FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
```

Comment:
@joelmartinez this is an example where I'm seeing the fxa values being added. Is it really needed?

Reply:
It's not needed, and in the next release we should see these go away … but AFAIK, they shouldn't cause any issues either if you leave them.